README.md

```diff
@@ -1,29 +1,288 @@
+> - Parent User need: https://github.com/stefpiatek/dummy-medical-software/issues/5
+>
+> Would you like to increment the version? (y/n): <prompt response from user - y>
+> INFO: Updated tag to "qw-v2"
+> INFO: There are 2 design outputs that link to this Design Input, please ensure one of them has the "qw-v2" tag if it resolves the updated information
+>
+> - https://github.com/stefpiatek/dummy-medical-software/pull/7
+> - https://github.com/stefpiatek/dummy-medical-software/pull/12
+>   ...
+>   INFO: :heavy_check_mark: 12 new QW items added to state
+>   INFO: :heavy_check_mark: 47 QW items checked
+>   INFO: Please commit the changes in the ".qw" directory
+
+Example response when the change is trivial and does not warrant a change in the version of the QW item:
+
+> ...
+> Would you like to increment the version? (y/n): <prompt response from user - n>
+> INFO: Tag kept at "qw-v1", data for the Design Input has been updated to the current state
+> ...
+
+### Creating a documentation release
+
+When you're ready to update the documentation in your Quality Management System (QMS), you can use the QW tool's `release` command.
+Running this will:
+
+- Ensure that all issues and pull requests have been marked with `qw-ignore` or one of the `qw-` item tags, raising an error (and stopping) to ensure all items are tagged
+- Ensure that all QW items have versions
+- Ensure the entire chain `design validation -> design verification -> design output -> design input -> user need` is consistent with QW rules, starting from the furthest stage of QW items. So if there is no `design validation`, then the chain will start from `design verification`. If there was only a `user need` and `design input`s, then only these would be validated
+- Create word documents based on the QW template for export
+- [name=Stef] Optionally? Create an html page that shows a burndown graph for each of the QW item types, showing the number completed and outstanding over time. Would this be useful?
+
+```
+qw release qms_docs
+```
+
+> Creating a release for QW
+> Creating "qms_docs" directory if it doesn't already exist, and overwriting any previously exported files
+> INFO: :heavy_check_mark: 47 QW items checked
+> INFO: :heavy_check_mark: Documents have been written to the "qms_docs" directory
+
+# FAQ and common issues
+
+- Can I import an existing project into QW
+  - There's nothing stopping this, though each issue and pull request would need to match the required format, and be tagged appropriately by the time you'd like to run a release. This may be a reasonable amount of work and we expect issues may need to be created.
+-
+
+# Open questions
+
+- Which document parts are the most useful for export, and would we want the user to be able to edit the template (update styles and placeholder text?)
+- Would the burndown chart be useful to get an overview for development?
+- Gate who can authorise a PR, or can anyone?
```

Comment:
If we're gating PRs then it'd all be using github rules and codeowners files. Though I'm less sure QW should control this, and its more up to the users to make sure they comply with their own development process?

Reply:
This is an excellent idea. We should absolutely lean into github's natural way of doing things where they exist. Escpecially as gitlab does it too.
In this case our configuration should be split into many csv files (or similar) rather than one giant .json